include/igl/copyleft/comiso/miq.cpp

```diff
@@ -139,6 +139,10 @@ namespace comiso {
   template <typename DerivedV, typename DerivedF>
   class PoissonSolver
   {
+  private:
+
+    // Penalization term for integer variables used in mixedIntegerSolve
+    const double INT_PENALIZATION = 0.000001;
```

Comment:
I'd just keep the name "penalization". "int_penalization" sounds like it should be an integer.

Reply:
done